infrastructure/running-a-node/setup-full-node.md

```diff
@@ -1,4 +1,258 @@
+        ```
+
+        Finally, issue a reload to have your modifications applied by systemd:
+
+        ```bash
+        systemctl daemon-reload
+        ```
+
+### Use Docker
+
+Finally, you can use Docker to run your node in a container. Doing this is more advanced, so it's best left up to those already familiar with Docker or who have completed the other setup instructions in this guide. 
+
+Be aware that when you run Polkadot in Docker, the process only listens on `localhost` by default. If you would like to connect to your node's services (RPC and Prometheus), you need to ensure that you run the node with the `--rpc-external` and `--prometheus-external` commands similar to the following:
+
+```bash
+docker run \
+-p 9944:9944 \
+-p 9615:9615 parity/polkadot:v0.9.13 \
+--name "calling_home_from_a_docker_container" \
+--rpc-external \
+--prometheus-external
+```
+
+Use the following commands to make your node externally accessible as an archive or pruned node. The options and flags will be explained in the following sections.
+
+For an externally accessible Polkadot archive node run:
+
+```bash
+polkadot --chain polkadot \
+--name INSERT_YOUR_NODE_NAME \
+--state-pruning archive \
+--blocks-pruning archive \
+--rpc-cors all \
+--rpc-methods safe 
+```
+
+For an externally accessible Polkadot pruned node run:
+
+```bash
+polkadot --chain polkadot \
+--name INSERT_YOUR_NODE_NAME \
+--state-pruning 1000 \
+--blocks-pruning archive \
+--rpc-cors all \
+--rpc-methods safe
+```
+
+## Secure the RPC Server
+
+The node startup settings allow you to choose what to expose, how many connections to expose, and which systems should be granted access through the RPC server. Some settings you can configure to increase node security include:
+
+- **Limit usable methods** - with  the `--rpc-methods` flag. An easy way to set this to a safe mode is: 
+```bash
+--rpc-methods safe
+```
+- **Set maximum number of connections** - with the `--rpc-max-connections` flag. For example, to limit maximum connections to 200 use:
+```bash
+--rpc-max-connections 200
+```
+- **Set server access permissions** - with the `--rpc-cors` flag. By default, localhost and Polkadot.js can access the RPC server. To allow access from everywhere, you can use: 
+```bash
+--rpc-cors all
+```
+
+For a list of important flags when running RPC nodes, refer to the Parity DevOps documentation: [Important Flags for Running an RPC Node](https://paritytech.github.io/devops-guide/guides/rpc_index.html?#important-flags-for-running-an-rpc-node){target=\_blank}
+
+## Secure the WebSocket Port
+
+To securely access your WebSocket (WS) connection over an SSL-enabled connection (necessary for SSL-enabled developer consoles), you'll need to convert the WS connection to a secure WSS connection. You can complete this conversion using a proxy and an SSL certificate. For detailed steps on setting this up, refer to the [Set Up Secure WebSockets](TODO: add path){target=\_blank} guide.
```

Comment:
We're not going to have docs on Securing WebSockets, so this link can be removed